set-viewpoint-rotation/src/main/res/layout/activity_main.xml

```diff
@@ -7,48 +7,54 @@
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_alignParentBottom="true"
-            android:layout_centerHorizontal="true"
-            android:layout_marginBottom="68dp"
-            android:layout_marginLeft="16dp"
-            android:layout_marginRight="16dp"
-            android:layout_toStartOf="@+id/rotationValueText"
-            android:max="360"
-            app:layout_constraintBottom_toBottomOf="@+id/mapView"
-            app:layout_constraintLeft_toLeftOf="@+id/mapView"
-            app:layout_constraintRight_toLeftOf="@+id/rotationValueText" />
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/mapView">
 
-        <TextView
-            android:id="@+id/rotationValueText"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_alignParentEnd="true"
-            android:layout_alignTop="@+id/rotationSeekBar"
-            android:layout_marginEnd="16dp"
-            android:text="@string/initial_rotation_text"
-            android:textSize="18sp"
-            android:textStyle="bold"
-            android:textColor="@android:color/black"
-            app:layout_constraintBottom_toBottomOf="@+id/rotationSeekBar"
-            app:layout_constraintRight_toRightOf="parent"
-            tools:text="180" />
+            <SeekBar
+                android:id="@+id/rotationSeekBar"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="10dp"
+                android:layout_marginTop="20dp"
+                android:layout_marginEnd="40dp"
+                android:layout_marginBottom="20dp"
+                android:max="360"
+                android:progressBackgroundTint="@android:color/darker_gray"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toStartOf="@+id/rotationValueText"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/rotationValueText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/initial_rotation_text"
+                android:textColor="@color/textColor"
+                android:textSize="18sp"
+                android:textStyle="bold"
+                app:layout_constraintBottom_toBottomOf="@+id/rotationSeekBar"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintStart_toEndOf="@+id/rotationSeekBar"
+                app:layout_constraintTop_toTopOf="@+id/rotationSeekBar" />
```

Comment:
```suggestion
            <com.google.android.material.slider.Slider
                android:id="@+id/rotationSlider"
                android:layout_width="0dp"
                android:layout_height="wrap_content"
                android:layout_marginStart="16dp"
                android:stepSize="1"
                android:valueFrom="0"
                android:valueTo="360"
                app:labelBehavior="gone"
                app:layout_constraintBottom_toBottomOf="parent"
                app:layout_constraintEnd_toStartOf="@+id/rotationValueText"
                app:layout_constraintStart_toStartOf="parent"
                app:layout_constraintTop_toTopOf="parent"
                app:tickVisible="false" />

            <TextView
                android:id="@+id/rotationValueText"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_marginEnd="16dp"
                android:text="@string/initial_rotation_text"
                android:textColor="@color/textColor"
                android:textSize="18sp"
                android:textStyle="bold"
                app:layout_constraintBottom_toBottomOf="parent"
                app:layout_constraintEnd_toEndOf="parent"
                app:layout_constraintTop_toTopOf="parent" />
```

Seems like this is the only sample still using `SeekBar`, let's move to use `com.google.android.material.slider.Slider` for the sample. 

Also, noticed that `20dp` top and bottom padding is large, I've provided a suggestion to match the contraints of the layout to the size of the slider.